src/iam.ts

```diff
@@ -778,7 +778,7 @@ export class IAM extends IAMBase {
     newOwner: string;
     returnSteps?: boolean;
   }) {
-     newOwner = parseDID(newOwner);
+      newOwner = parseDID(newOwner);
     const orgNamespaces = [
```

Comment:
Fixing strange alignment
```suggestion
    newOwner = parseDID(newOwner);
    const orgNamespaces = [
```